app/src/main/java/org/fossasia/openevent/general/ticket/TicketViewHolder.kt

```diff
@@ -30,9 +30,15 @@ class TicketViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
             itemView.orderRange.adapter = ArrayAdapter(itemView.context, R.layout.select_dialog_singlechoice, spinnerList)
         }
 
+        val price = StringBuilder()
```

Comment:
why not store the currency in a different variable and reference it using $ in `  itemView.price.text`
using the stringBuilder makes it look a little complicated

Reply:
Updated